### ISSUE TITLE
RawGit has reached the end of its useful life

### ISSUE THREAD

**Issue by nikolai-b** [npct/pct-shiny#821]
https://cdn.rawgit.com is no more.  It suggests we move to another service like GitHub pages (but that is limited to 1 GB files so is probably not appropriate).  [S3](https://aws.amazon.com/s3/) is probably a very sensible place to store files (and would mean we could have revisions) but that would be quite a bit of work.

The easiest thing would be to use https://www.jsdelivr.com/rawgit (hopefully they won't shut down, looks like it has sponsors but might need to check it more) as we can simply change all the URL and everything should _Just Work_ ™

**Comment by Robinlovelace** [comment]
What about GitHub releases?

**Comment by Robinlovelace** [comment]
Or the `raw` URLs?

**Comment by nikolai-b** [comment]
The raw URLs would reintroduce https://github.com/npct/pct-shiny/issues/621
I'm not sure about github releases, would you have the files would be stored outside the repo? It looks possible but I've not investigated what the full flow would be like. The advantage of RawGit was we didn't need to **do** anything (which could easily be forgotten). Would you be happy with jsdelivr?

**Comment by Robinlovelace** [comment]
#621 is pretty minor and just using the raw urls like

- https://media.githubusercontent.com/media/npct/pct-outputs-regional-notR/master/commute/msoa/west-midlands/z.geojson

and 

- https://github.com/npct/pct-outputs-regional-notR/raw/master/commute/msoa/west-midlands/od_attributes.csv

Work fine. Most people will download the GeoJSON option, see the raw .csv files in Firefox isn't that but and simplicity is good. There may be advantages of 'jsdelivr' that are outweigh the benefits of simplicity, and I'm hesitant to recommend another service outside GH but open-minded about the best way forward.

**Comment by Robinlovelace** [comment]
Test case: 

https://cdn.rawgit.com/npct/pct-shiny/master/regions_www/www/static/03a_manual/pct-bike-eng-user-manual-a.pdf

vs

https://github.com/npct/pct-shiny/raw/master/regions_www/www/static/03a_manual/pct-bike-eng-user-manual-a.pdf

and 

https://cdn.rawgit.com/npct/pct-outputs-regional-notR/commute/msoa/west-midlands/od_attributes.csv

vs

https://github.com/npct/pct-outputs-regional-notR/raw/master/commute/msoa/west-midlands/od_attributes.csv

**Comment by Robinlovelace** [comment]
```
sd 'https://cdn.rawgit.com/npct/pct-shiny/master/' 'https://github.com/npct/pct-shiny/raw/master/`
```

Should do it.

**Comment by AnnaGoodman1** [comment]
fwiw I don't think we ever did fix #621, it has always been that way with me. So what @Robinlovelace suggests sounds ok to me if @nikolai-b also happy

**Comment by Robinlovelace** [comment]
Great, please merge @AnnaGoodman1 or @nikolai-b

**Comment by nikolai-b** [comment]
Fixed in #824
